queries.sql

```diff
@@ -20,4 +20,12 @@ SELECT MAX(weight_kg) FROM animals WHERE species IN ('digimon');
 SELECT MIN(weight_kg) FROM animals WHERE species IN ('pokemon');
 SELECT MAX(weight_kg) FROM animals WHERE species IN ('pokemon');
 SELECT id, name, date_of_birth, SUM(escape_attempt), COUNT(escape_attempt), SUM(escape_attempt) / COUNT(escape_attempt) as average FROM animals WHERE species IN ('pokemon') AND date_of_birth >= '12/05/1990'::date AND date_of_birth <= '01/01/2021'::date GROUP BY id, name, date_of_birth;
```

Comment:
- [ ] In the data.sql file, students were instructed to add data to the `owners` and `species` table. For example;
 ```
 INSERT INTO owners (full_name, age)
 VALUES ('Sam Smith', 34);

```
```
 INSERT INTO species (name)
 VALUES ('Pokemon');

```

Reply:
`owners data and species data` have been inserted already.